File: Iniciante/1040.cpp
#include <iostream>
#include <iomanip>
using namespace std;

int main() {
    double n1, n2, n3, n4;
    cin >> n1 >> n2 >> n3 >> n4;
    std::cout << std::fixed << std::setprecision(1);
    
    double media;
    media = (n1 * 2 + n2 * 3 + n3 * 4 + n4 * 1)/10;
    cout << "Media: "<< media << endl;
    
    if(media >= 7)
        cout << "Aluno aprovado."<< endl;
    
    else if(media < 5)
        cout << "Aluno reprovado."<< endl;
    
    else{
        cout << "Aluno em exame."<< endl;
        double exame;
        cin >> exame;
        cout << "Nota do exame: "<< exame << endl;
        if((media + exame)/2 >= 5)
            cout << "Aluno aprovado."<< endl;
        
        else
            cout << "Aluno reprovado."<< endl;
            
        cout << "Media final: "<< (media + exame)/2 << endl;
    }
    
    return 0;
}

File: Iniciante/C#/1040.cs
using System;
class URI {
    static void Main(string[] args) { 
        string[] input = Console.ReadLine().Split(' ');
        double n1 = double.Parse(input[0]);
        double n2 = double.Parse(input[1]);
        double n3 = double.Parse(input[2]);
        double n4 = double.Parse(input[3]);
        
        double media = (n1 * 2 + n2 * 3 + n3 * 4 + n4 * 1)/10;
        Console.WriteLine("Media: " + Math.Round(media, 1).ToString("0.0"));
        
        if(media >= 7)
            Console.WriteLine("Aluno aprovado.");
        else if(media < 5)
            Console.WriteLine("Aluno reprovado.");
        else {
            double exame = double.Parse(Console.ReadLine());
            Console.WriteLine("Aluno em exame.");
            Console.WriteLine("Nota do exame: " + Math.Round(exame, 1).ToString("0.0"));
            if((media + exame)/2 >= 5)
                Console.WriteLine("Aluno aprovado.");
            else
                Console.WriteLine("Aluno reprovado.");
            Console.WriteLine("Media final: " + Math.Round((media + exame)/2, 1).ToString("0.0"));
        }
    }
}